gc/base/standard/Scavenger.cpp

```diff
@@ -981,6 +981,19 @@ MM_Scavenger::calcGCStats(MM_EnvironmentStandard *env)
 		uintptr_t initialFree = env->_cycleState->_activeSubSpace->getActualActiveFreeMemorySize();
 		uintptr_t tenureAggregateBytes = 0;
 		float tenureBytesDeviation = 0;
+		uintptr_t survivorAllocated = 0;
+
+		if (IS_CONCURRENT_ENABLED) {
+			/* InitialFree is more pricisely meant to represent the amount of memory that is predicted to be allocated between 2 cycles,
+			 * that is later used of CM kickoff. For simple STW Scavenge, it is just indeed inital free memory at the start of allocation (end of a cycle).
+			 * But for CS, we also have to account for the memory allocated during active CS cycle (allocated from surivior/allocate hybrid area).
+			 * While initialFree is based on the predicted future allocation, survivorAllocated is acutally what was allocated from just completed cycle,
+			 * since we cannot predict how much it will be allocated (due to other allocations from GC). But that 1-cycle-off difference should be ok,
+			 * since we anyway historically average these values on effectively a range of a few cycles.
+			 */
```

Comment:
fixed typos and formatting
```suggestion
			/* InitialFree is more precisely intended to represent the amount of
 			 * memory predicted to be allocated between two cycles, which is later
 			 * used for CM kickoff. For a simple STW Scavenge, it is indeed just the
  			 * initial free memory at the start of the allocation phase (end of a
  			 * cycle). However, for Concurrent Scavenge (CS), we also have to account
 			 * for memory allocated during the active CS cycle (allocated from the
  			 * survivor/allocate hybrid area). While InitialFree is based on predicted
  			 * future allocation, survivorAllocated represents what was actually
  			 * allocated in the just-completed cycle, since we cannot predict how much
  			 * will be allocated (due to other allocations by GC). That one-cycle-off
  			 * difference should be acceptable, as we historically average these values
  			 * over a span of a few cycles anyway.
  			 */
```